appuio/haproxy/templates/configmap-galera-checkscript.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if or (eq .Values.haproxy.config "galera") (eq .Values.haproxy.config "galerak8s") }}
+
+kind: ConfigMap
+apiVersion: v1
+data:
+  script.sh: "#!/bin/bash\n\n# Run the MySQL query and store the result in a Bash
```

Comment:
Can't helm read from a file and insert the string here?

Having the script separate in a readable form would greatly help the reviewability/maintainability of this script.

Reply:
+1 on that. I would try to do this via file if possible.

Reply:
As a compromise I fixed formatting - wdyt?

Reply:
A separate file would make it easier to check with shellcheck.

There's a way to read files in helm: https://helm.sh/docs/chart_template_guide/accessing_files/

Something like this should do the trick:
```yaml
data:
  script.sh: |-
        {{ .Files.Get "script.sh" }}
```